loader/dd_library_loader.c

```diff
@@ -0,0 +1,548 @@
+/* dd_library_loader extension for PHP */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include <Zend/zend_extensions.h>
+#include <php.h>
+#include <php_ini.h>
+#include <stdbool.h>
+#include <ext/standard/basic_functions.h>
+
+#include "compat_php.h"
+#include "php_dd_library_loader.h"
+
+static bool debug_logs = false;
+static bool force_load = false;
+static char *telemetry_forwarder_path = NULL;
+static char *package_path = "/home/circleci/app/dd-library-php";  // FIXME
```

Comment:
Is this something to hardcode, or can we look it up in configuration_hash (`cfg_get_string`)?
Like, from my perspective, we anyway always define an INI file to include the `zend_extension`. So at that place, just indicate the package_path as well I'd say.

Reply:
I see you anyway have an env var. Can we just require it to be set?

Reply:
Done. Env var is now required.